javascript/javascript1/week1/Ez Namey/startup_name.js

```diff
@@ -0,0 +1,37 @@
+const firstWords = [
+  "Hive",
+  "Vibe",
+  "Cloud",
+  "Green",
+  "Sync",
+  "Neo",
+  "Bright",
+  "Spark",
+  "Urban",
+  "Swift",
+];
+
+const secondWords = [
+  "Edge",
+  "Nest",
+  "Venture",
+  "Link",
+  "Pulse",
+  "Path",
+  "Wave",
+  "Flow",
+  "Bridge",
+  "Tech",
+];
+
+const randomNumber = Math.floor(Math.random() * 10);
```

Comment:
We can use two random numbers to get truly random names :) Right now the names will always come in parallel.

Reply:
Updated the code with two random numbers